nexus/examples/config.toml

```diff
@@ -39,7 +39,7 @@ external_dns_servers = ["1.1.1.1", "9.9.9.9"]
 
 [deployment.dropshot_external]
 # IP Address and TCP port on which to listen for the external API
-bind_address = "127.0.0.1:12220"
+bind_address = "0.0.0.0:12220"
```

Comment:
This feels kind of dangerous because we might be exposing this service on any IP on the developer's network, right?

Can your use case provide its own config file for this instead?  Or maybe make this an argument to `omicron-dev`?